packages/scandipwa/src/component/CheckoutPayment/CheckoutPayment.style.scss

```diff
@@ -44,7 +44,16 @@
         display: flex;
         align-items: flex-start;
         font-size: 14px;
-        cursor: pointer;
+
+        &_noCheckbox {
+            .Field [type='checkbox']:disabled {
+                opacity: 0;
```

Comment:
When you hide the element with the `opacity: 0`, you preserve its space.
Do you need it there? Or maybe it's better to hide it completely and make a space with the padding instead because padding size is more controllable than transparent element size?